01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env ruby
+
+range = 1..20
+range.each do |num|
```

Comment:
変数に入れずに直接eachしてもいいかもしれません。

```suggestion
(1..20).each do |num|
```

あと、`num`は`n`と略すことが多いかも。


```suggestion
(1..20).each do |n|
```

とはいえ、オレオレ略語は混乱の原因なので、乱用にはご注意を。
https://blog.jnito.com/entry/2020/10/20/092724

Reply:
ありがとうございます！修正しました！